_drafts/2017-06-20-free-friday.md

```diff
@@ -0,0 +1,75 @@
+---
+title:  "Free friday for all - Chapters in Sense/Net"
+author: borsi
+tags: [community, developers, communication, company culture, freefriday]
+---
+In their [2004 IPO letter][64bbc0a2], founders Larry Page and Sergey Brin highlighted their idea of the famous "Google 20%". A lot of time has passed since then and the legend faded too: is it really 20% of their time, or is it more like "Google 120%"?
+
+  [64bbc0a2]: https://abc.xyz/investor/founders-letters/2004/ipo-letter.html "Google IPO letter"
+
+Nontheless, we wanted to empower our people so we decided to give it - or at least **a** version of it - a go.
+---
+
+> We encourage our employees, in addition to their regular projects, to spend 20% of their time working on what they think will most benefit Google. This empowers them to be more creative and innovative. Many of our significant advances have happened in this manner. For example, AdSense for content and Google News were both prototyped in “20% time.” Most risky projects fizzle, often teaching us something. Others succeed and become attractive businesses.
+
+They wrote. It almost sounds too good to be true: our people do what they think is best for us, and it truly gets better. It is always important to remind ourselves before getting overly excited about things like this, that _[we are not Google][bfca94a4]_. So before rushing to the solution, we need to think about our on corporate structure, and problems we need solving.
+
+  [bfca94a4]: https://blog.bradfieldcs.com/you-are-not-google-84912cf44afb "You are not google - Ozan Onay"
+
+### How Sense/Net Inc. works
+
+Sense/Net Inc. - the company behind sensenet ECM - has two products. One actual product (sensenet ECM) and a solution integrator service: since we know our product the best, we can create unique solutions for companies all around the world (we also do training and quite a lot of support, but let's just consider these parts of one or both of the products I just mentioned).
+
+This results in teams that not just work on different projects, they do **different kinds of things**. In time, people end up with vastly different know-hows and knowledge levels. It is hard to admit, but _sometimes_ there are multiple solutions for the same problem. Or worse, there are the same solutions implemented several times for the same problem.
```

Comment:
This has a different meaning. I want to say that the teams (that do things differently) are the result of the causes.